examples/apps/attributable-map/package.json

```diff
@@ -11,6 +11,24 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"exports": {
+		".": {
+			"default": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			}
+		},
+		"./internal": {
+			"import": {
+				"types": "./lib/index.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.cjs"
+			}
```

Comment:
In general you want to have an import and require section in each export except in cases where the package is only building one module type.

Reply:
How can I tell if there is only one module type?